apps/web/src/lib/components/ui/community-details/libraryDetails.svelte

```diff
@@ -1,31 +1,21 @@
 <script lang="ts">
 	import * as Dialog from '$lib/components/ui/dialog';
-	import { User } from 'lucide-svelte';
 	import { createEventDispatcher } from 'svelte';
-	import dayjs from 'dayjs';
-	import relativeTime from 'dayjs/plugin/relativeTime';
 	import type { Agent, Crew } from '$lib/types/models';
-
-	dayjs.extend(relativeTime);
+	import { AgentDetails, CrewDetails } from '$lib/components/ui/community-details';
 
 	export let type: 'agent' | 'crew';
-	export let displayedAgent: Agent | Crew;
+	export let displayedAgent: Agent | Crew
 	export let showDetails: boolean;
+
 	const dispatch = createEventDispatcher();
-	dayjs.extend(relativeTime);
 
 	function closeDialog() {
 		dispatch('close');
 		showDetails = false;
-		console.log(showDetails);
-	}
-
-	function timeSince(dateIsoString: Date | string | number) {
-		return dayjs(dateIsoString).fromNow(true);
 	}
 
 	$: isAgent = type === 'agent';
```

Comment:
all this logic here seems unnecessary since you're already checking the type in the component below. (line 29)